lib/dor/datastreams/workflow_ds.rb

```diff
@@ -9,7 +8,7 @@ class WorkflowDs < ActiveFedora::OmDatastream
         t.process {
           t.name_(    :path => {:attribute => 'name'     }, :index_as => [:displayable, :not_searchable] )
           t.status(   :path => {:attribute => 'status'   }, :index_as => [:displayable, :not_searchable] )
-          t.timestamp(:path => {:attribute => 'datetime' }, :index_as => [:displayable, :not_searchable] ) # , :data_type => :date)
+          t.timestamp(:path => {:attribute => 'datetime' }, :index_as => [:displayable, :not_searchable] ) #, :data_type => :date)
```

Comment:
Has a library change (ActiveFedora?) made the extra `:data_type` arg obsolete and/or unnecessary?  Why leave the comment?